src/Tests/CreateOrderTest.php

```diff
@@ -131,82 +71,128 @@ public function test_get_property()
         );
     }
 
-    public function test_exception_validation()
+    public function getTransactions()
+    {
+        return new Transaction(
+            paymentMethod: 'ideal',
+            paymentMethodDetails: new PaymentMethodDetails(
+                issuer_id: "15"
+            )
+        );
+    }
+
+    public function getCustomer()
+    {
+        return new Customer(
+            additionalAddresses: new AdditionalAddresses(
+                new Address(
+                    addressType: 'customer',
+                    postalCode: '12345',
+                    street: 'Soborna',
+                    city: 'Poltava',
+                    country: new Country(
+                        'UA'
+                    )
+                ),
+                new Address(
+                    addressType: 'billing',
+                    postalCode: '1234567',
+                    street: 'Donauweg',
+                    city: 'Amsterdam',
+                    country: new Country(
+                        'NL'
+                    ),
+                    housenumber: "10"
+                )
+            ),
+            firstName: 'Alexander',
+            lastName: 'Tiutiunnyk',
+            emailAddress: new EmailAddress(
+                'tutunikssa@gmail.com'
+            ),
+            gender: 'male',
+            phoneNumbers: new PhoneNumbers(
+                '0951018201'
+            ),
+            merchantCustomerId: '15',
+            birthdate: new \GingerPluginSdk\Properties\Birthdate('1999-09-01'),
+            locale: new Locale(
+                'Ua_ua'
+
+            )
+        );
+    }
+
+    public function getOrderLines()
+    {
+        return new OrderLines(
+            new Line(
+                type: 'physical',
+                merchantOrderLineId: "5",
+                name: 'Milk',
+                quantity: 1,
+                amount: 1.00,
+                vatPercentage: 50,
+                currency: new Currency(
+                    'EUR'
+                )
+            )
+        );
+    }
+
+    public function getClient()
+    {
+        return  new \GingerPluginSdk\Entities\Client(
+            userAgent: $_SERVER['HTTP_USER_AGENT'],
+            platformName: 'docker',
+            platformVersion: '1',
+            pluginName: 'ginger-plugin-sdk',
+            pluginVersion: '1.0.0'
+        );
+    }
```

Comment:
This functions contain common data for order. If this idea look acceptable, then probably it should be moved to new class (for example "HelperOrderData")